gusb/gusb-device.c

```diff
@@ -2199,9 +2211,11 @@ g_usb_device_control_transfer_async(GUsbDevice *self,
 						event_id);
 			return;
 		}
-		if (g_usb_device_event_get_status(event) != LIBUSB_TRANSFER_COMPLETED) {
-			g_usb_device_libusb_status_to_gerror(g_usb_device_event_get_status(event),
-							     &error);
+		if (!g_usb_device_libusb_error_to_gerror(self,
```

Comment:
is g_usb_device_libusb_error_to_gerror setting the GError if it fails?

Reply:
If rc == LIBUSB_SUCCESS then we return TRUE, i.e. don't set an error.

Reply:
ok, looks good then